### DISCUSSION TITLE
Get 404 when trying to obtain the odds for the event which is in TRADING status

### DISCUSSION THREAD

**Discussion by fdan1c** [cloudbet/docs#58] [Q&A]
Could you, please, help me with the following issue.

I'm trying to build the app which obtains the events currently in TRADING status. A week ago or so I started to get HTTP/404 for the odds request for such events which are in TRADING status.

As one of the examples: using GET {{baseUrl}}/v2/odds/fixtures?sport=soccer&date=2023-03-26&limit=1000 I' getting the response which contains event with id=17392141, which is in TRADING status. But when I'm trying to request the odds for this event I get HTTP/404 which is so weird. Why it happens? Are there any limitations placed to my account? There was no such behavior 2 weeks ago. The issue even doesn't relate to placing the bets, but to getting the event odds only.

It can't be my code issue, since a portion of other events odds are returned successfully using the same code. 
Please, advise.

**Comment by kgravenreuth** [comment]
Hi fdan1c,

Thanks for reporting this issue. We will review the offering of events here on the `v2/odds/fixtures` endpoint, while you can take the 404 to mean that the specific event is not available for betting at the moment. 

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
